azure-client-runtime/src/test/java/com/microsoft/azure/v2/AzureProxyToRestProxyTests.java

```diff
@@ -447,47 +447,24 @@ public void SyncPutRequestWithUnexpectedResponseAndExceptionType() {
     private interface Service10 {
         @HEAD("anything")
         @ExpectedResponses({200})
-        HttpBinJSON head();
-
-        @HEAD("anything")
-        @ExpectedResponses({200})
         void voidHead();
 
         @HEAD("anything")
         @ExpectedResponses({200})
-        Single<HttpBinJSON> headAsync();
-
-        @HEAD("anything")
-        @ExpectedResponses({200})
         Completable completableHeadAsync();
     }
 
     @Test
-    public void SyncHeadRequest() {
-        final HttpBinJSON json = createService(Service10.class)
-                .head();
-        assertNull(json);
-    }
```

Comment:
Why are we getting rid of some of these tests?

Reply:
`Observable<Void>` and `Single<Void>` are now illegal, so I'll bring this back as `Single<RestResponse<Void, Void>>`. I think I did that in the other tests, but not here..